hoomd/md/pair/pair.py

```diff
@@ -1165,6 +1165,83 @@ def __init__(self, nlist, r_cut=None, r_on=0., mode='none'):
         self._add_typeparam(params)
 
 
+class ExpandedMie(Pair):
+    r"""Expanded Mie pair potential.
+
+    Args:
+        nlist (`hoomd.md.nlist.NList`): Neighbor list
+        r_cut (float): Default cutoff radius (in distance units).
+        r_on (float): Default turn-on radius (in distance units).
```

Comment:
```suggestion
        r_cut (float): Default cutoff radius :math:`[\\mathrm{length}]`.
        r_on (float): Default turn-on radius :math:`[\\mathrm{length}]`.
```
Use the new-style units formatting from #1019

Reply:
Apply the same styling to epsilon, sigma, delta, n, and m below.